index.ts

```diff
@@ -1,26 +1,39 @@
-require('dotenv').config();
+import dotenv from 'dotenv'
 
-const fs = require('fs');
-const Discord = require('discord.js');
-const { prefix, botAdmin, botOwner } = require('./config.json');
+dotenv.config();
+
+import * as fs from 'fs';
+import * as Discord from 'discord.js';
+import { prefix, botAdmin, botOwner } from './config.json';
+
+interface Command {
+	name: string,
+	description: string,
+	aliases: string[],
+	cooldown: number,
+	execute(message: any): Function,
+}
 
 const intents = new Discord.Intents(['GUILDS', 'GUILD_BANS', 'GUILD_MESSAGES', 'GUILD_PRESENCES', 'DIRECT_MESSAGES']);
 const client = new Discord.Client({
 	ws: { intents: intents },
 	presence: { activity: { name: 'Costpap shout', type: 'LISTENING' }, status: 'online' },
 });
+
 client.commands = new Discord.Collection();
 const cooldowns = new Discord.Collection();
 
 const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'));
 
+const importCommand = async (command: any): Promise<Command> => return (await import(`./commands/${command}`) as Command);
+
 for (const file of commandFiles) {
-	const command = require(`./commands/${file}`);
+	const command = importCommand(file);
 	client.commands.set(command.name, command);
 }
 
-if (!process.env.TOKEN) return console.error('Missing client token. Shutting down...');
-if (!prefix || prefix.length > 5) return console.error('Prefix is either missing or too long. Shutting down...');
+if (!process.env.TOKEN) console.error('Missing client token. Shutting down...');
+if (!prefix || prefix.length > 5) console.error('Prefix is either missing or too long. Shutting down...');
```

Comment:
```suggestion
if (!prefix || prefix.length > 5) return console.error('Prefix is either missing or too long. Shutting down...');
```
Shouldn't this include `return`? The purpose of this line is to stop anything from being executed if there is no prefix provided or if the provided prefix is too long.